stories/Templates/PublicationPage/PublicationPage.jsx

```diff
@@ -41,94 +41,128 @@ export const PublicationPage = ({
+                        <P label={paragraphTop3} />
+                    </div>
+                    <div class="column publication-menu">
+                        <div className='grid-x grid-margin-x'>
+                            <div class="cell small-4">
+                                <Heading type='6' label={menuTitle1}/>
+                            </div>
+                            <div class="cell small-8">
+                                <Menu data={menuData1}/>
+                            </div>
+                        </div>
+                        <div className='grid-x grid-margin-x'>
+                            <div class="cell small-4">
+                                <Heading type='6' label={menuTitle2}/>
+                            </div>
+                            <div class="cell small-8">
+                                <Menu data={menuData2}/>
+                            </div>
+                        </div>
+                        <div className='grid-x grid-margin-x'>
+                            <div class="cell small-4">
+                                <Heading type='6' label={menuTitle3}/>
+                            </div>
+                            <div class="cell small-8">
+                                <Menu data={menuData3}/>
+                            </div>
+                        </div>
+                        <div className='grid-x grid-margin-x'>
+                            <div class="cell small-4">
+                                <Heading type='6' label={menuTitle4}/>
+                            </div>
+                            <div class="cell small-8">
+                                <Menu data={menuData4}/>
+                            </div>
+                        </div>
+                    </div>                    
+                </div>
             </div>
-          </div>
-          <div className="cell small-12 large-6 large-offset-6">
-            <div className="column margin-top-36">
-              <CtaButton label={buttontype} Type="Secondary" />
+            <div class="grid-x grid-margin-x margin-top-48 related-publication">
+                <div class="cell small-12 large-6 large-offset-6 border-top-2">
+                    <div class="column">
+                        <Heading 
+                            type='2' 
+                            label={relatedPublicationHeading} 
+                        />
+                    </div>
+                </div>
+                <div class="cell small-12 large-6 large-offset-6">
+                    <div class="column">
+                        <ContentCardWithOutImage className="card-out"
+                            Emphasize='false'
+                            data={ContentCardWithOutImageData}
+                        />
+                    </div>
+                </div>
+                <div class="cell small-12 large-6 large-offset-6">
+                    <div class="column margin-top-36">
```

Comment:
@neelesh-tippe Please check properly it should be className and you are using class in the complete file